pages/docs/tutorials/native/mapping-strings-from-c.md

```diff
@@ -248,7 +248,7 @@ fun main() {
 
     val useMe2 = copy_string(pinned.addressOf(0), pinned.get().size - 1)
     if (useMe2 != 0) throw Error("Failed to read string from C")
-    pinned.get().stringFromUtf8()
+    pinned.get().decodeToString()
```

Comment:
There was a subtle difference between `stringFromUtf8` and `decodeToString()` around null-terminated strings. The former stops decoding on zero byte, whereas the latter decodes it normally as `nul` char.
The equivalent replacement would be `toKString()` from `kotlinx.cinterop` package.